### PR TITLE
ReadMe demo version

### PR DESCRIPTION
**Pull request by KimNikita**

### CODE REVIEW

**Comment by KimNikita** [comment]
issue #44

**Comment by KimNikita** [comment]
@DumDereDum  check pls

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
\ No newline at end of file
+# Magic-Piano
+
+Magic Piano is a computer vision-based project that allows you to play a virtual piano using hand recognition through a camera.
+
+## Pipeline
+
+### 1. Hand detection
+
+We get a frame from the camera and use Open-CV Hand Detector to recognize hands and get information about key points of the palm.
```
Comment (DumDereDum) [inline README.md]: We use OpenCV for image capturing, hand tracking is mediapipe

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+## How to use
+
+First, you need to clone this repository to a folder that does not have Russian letters in the path.
+Example:
+```bash
+cd C:/Users
+git clone https://github.com/DumDereDum/Magic-Piano.git
+```
+Then, go inside the folder "Magic-Piano" and run .exe file
```
Comment (DumDereDum) [inline README.md]: I guess that it is better to write about running code, like download pip libs, run main.py, or something like this

**Comment by KimNikita** [comment]
названия файлов .exe .py пока не пишу

**Comment by DumDereDum** [comment]
👍

**Pull request merged**